filters-share/specific_URL.txt

```diff
@@ -431,7 +430,6 @@
 ||yeongnam.com/web/js/statAd.js
 ||yeongnam.com/mnt/webdata/ad/
 ||5donews.co.kr/layouts/5donews/images/banner/
-||womennews.co.kr/adManager/css/adManager.css
```

Comment:
This blocks advertisement on http://www.womennews.co.kr/.

Reply:
This entry exists at https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L435 and https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L439